app/Models/Post.php

```diff
@@ -8,8 +8,19 @@
 class Post extends Model
 {
     use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
+        'title',
+        'body',
+    ];
+
     
     public function user(){
-        return $this->belongsTo('App\User');
+        return $this->belongsTo(User::class,'user_id');
```

Comment:
return $this->belongsTo(User::class, 'user_id');
Sau , phải có 1 dấu cách em nhé